control/cli.py

```diff
@@ -960,14 +961,15 @@ def listener_list(self, args):
                 for l in listeners_info.listeners:
                     adrfam = GatewayEnumUtils.get_key_from_value(pb2.AddressFamily, l.adrfam)
                     adrfam = self.format_adrfam(adrfam)
-                    listeners_list.append([l.host_name, l.trtype, adrfam, f"{l.traddr}:{l.trsvcid}"])
+                    secure = "Yes" if l.secure else "No"
```

Comment:
This should be human readable so I think it should use simple English. "Yes" and "No" are much more understood by non-programmers. If instead of a text format a Json one is used, you will get the True and False values.